src/addons/transitions/ReactTransitionGroup.js

```diff
@@ -38,6 +39,7 @@ var ReactTransitionGroup = React.createClass({
 
   getInitialState: function() {
     return {
+      // TODO: can we get useful debug information to show at this point?
```

Comment:
Interestingly, ReactInstanceMap doesn't have information for the component at this point. That's probably because the component hasn't been mounted.